core/src/main/scala/kafka/server/FetcherState.scala

```diff
@@ -32,21 +32,13 @@ object FetcherState {
     override protected def hasRateAndTimeMetric: Boolean = false
   }
 
-  case object AddPartitions extends FetcherState {
+  case object ModifyPartitionsAndGetCount extends FetcherState {
     def value = 1
   }
 
-  case object RemovePartitions extends FetcherState {
-    def value = 2
-  }
-
-  case object GetPartitionCount extends FetcherState {
-    def value = 3
-  }
-
```

Comment:
Just to be sure:  these are all purely internal to the server with no possibility of different-versioned components interacting with each other?  (That is, they're not part of an API that would have compatibility issues due to this otherwise backward-incompatible change, right?)

Reply:
That's right. These are purely internal to the server and NOT exposed to clients or any other service.